src/bindist/installdos

```diff
@@ -3,6 +3,8 @@
 SOURCE="$1"
 DRIVE_ROOT="$2"
 
+PROPRIETARY_DOS_INSTALLED_EXIT_CODE=100
```

Comment:
Oh.
Would you mind using popen() instead of
using return codes for that?
You'll have to use popen anyway, because
what the script prints, should be visible on
dosemu window (for example if you do --list,
or the installation status).